command/rm.go

```diff
@@ -165,14 +171,14 @@ func sourcesHaveSameType(sources ...string) error {
 		if hasLocal && hasRemote {
 			return fmt.Errorf("arguments cannot have both local and remote sources")
 		}
-	}
-	return nil
-}
-
-func validateRMCommand(c *cli.Context) error {
-	if !c.Args().Present() {
-		return fmt.Errorf("expected at least 1 object to remove")
+		if i == 0 {
+			firstBucket = srcurl.Bucket
+			continue
+		}
+		if srcurl.Bucket != firstBucket {
+			return fmt.Errorf("one rm command cannot be used for object removal of more than one bucket")
```

Comment:
how about this? `removal of objects with different buckets in a single command is not allowed`